package/boot/uboot-tools/uboot-envtools/files/qualcommax_ipq50xx

```diff
@@ -8,6 +8,11 @@ touch /etc/config/ubootenv
 board=$(board_name)
 
 case "$board" in
+glinet,gl-b3000)
+	idx="$(find_mtd_index 0:APPSBLENV)"
+	[ -n "$idx" ] && \
+		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x40000" "0x20000" "2"
```

Comment:
You dont need to set the number of NAND blocks, envtools can autodetect, so just drop the last `"2"` entry